studio/components/layouts/ProjectLayout/LayoutHeader/NotificationsPopover/index.tsx

```diff
@@ -33,9 +33,9 @@ const NotificationsPopover = () => {
 
   const [targetNotification, setTargetNotification] = useState<Notification>()
 
-  if (!notifications) return <></>
+  if (!notifications || !Array.isArray(notifications)) return <></>
```

Comment:
Potential simplification: `return null` instead of an empty fragment

Reply:
just realised that this looks weird too actually - shouldn't be hiding the notifications button when there's no notifications, but i'll handle this in future error handling refactor PRs